app/assets/javascripts/index.js

```diff
@@ -254,25 +254,27 @@ $(document).ready(function () {
           bottom: bbox.getSouth() - 0.0001
         };
 
-    if (object && object.type !== "note") query.select = object.type + object.id; // can't select notes
-    sendRemoteEditCommand(remoteEditHost + "/load_and_zoom?" + Qs.stringify(query), function () {
+    if (object && object.type !== "note") query.select = object.type + object.id; // It's not possible to select notes with this parameter.
+    sendRemoteEditCommand(remoteEditHost + "/load_and_zoom?" + Qs.stringify(query), true, function () {
       if (object && object.type === "note") {
         var noteQuery = { url: osmHost + OSM.apiUrl(object) };
-        sendRemoteEditCommand(remoteEditHost + "/import?" + Qs.stringify(noteQuery));
+        sendRemoteEditCommand(remoteEditHost + "/import?" + Qs.stringify(noteQuery), false); // This is an enhancement, don't display an error if it fails.
```

Comment:
There's no need for the first part of this comment - the part after the comma is the only bit that is relevant. I'd also probably say `on failure` rather than `if it fails` because it avoids questions of what exactly `it` refers to.